src/v/config/configuration.cc

```diff
@@ -512,6 +512,30 @@ configuration::configuration()
       "refreshed",
       {.visibility = visibility::tunable},
       2s)
+  , kafka_tcp_keepalive_idle_timeout_seconds(
+      *this,
+      "kafka_tcp_keepalive_timeout",
```

Comment:
This corresponds to `tcp_keepalive_time` in `/proc/sys, right? So this will be the usual period between keepalive packets when everything is working well, right?

Reply:
Yes correct. Yeah I wasn't sure whether I should basically give more details of how TCP keepalive works in those config descriptions or not. 

I guess it doesn't hurt.

Reply:
Yeah I think it doesn't hurt also especially since the kernel/socket tunable names are already confusing and no doubt will bite someone so we can kind of undo that problem with a better description.